### PR TITLE
Updated outdated commands

### LINKED ISSUE #3344: Feedback on page: /guides/v2.2/install-gde/prereq/file-system-perms.html
# General issue

Outdated commands recommended.

## Description:

Under "Step 2: Find the web server user’s group" the guide recommends to run the following command to get the apache user and group from the httpd.conf file:

```
egrep -i '^user|^group' /etc/httpd/conf/httpd.conf
```

While this is technically correct the `egrep` program has been deprecated. It is only available for historical reasons and backwards compatibility. The man page says: 

> In addition, two variant programs egrep and fgrep are available.  egrep is the same as  grep -E.   fgrep  is  the  same  as  grep -F.   Direct
       invocation as either egrep or fgrep is deprecated, but is provided to allow historical applications that rely on them to run unmodified.

## Possible solutions:

People should be encouraged to use `grep -E` instead, so the line should be changed to one of the following:

```
grep -E -i '^user|^group' /etc/httpd/conf/httpd.conf
# or
grep -Ei '^user|^group' /etc/httpd/conf/httpd.conf
```
Either of these two will work.

### PR DESCRIPTION
**Pull request by gelanivishal**
## This PR is a:

- [ ] New topic
- [X] Content update
- [X] Content fix or rewrite
- [X] Bug fix or improvement

## Summary
I have updated outdated commands. Fixes #3344. 

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.2/install-gde/prereq/file-system-perms.html
- https://devdocs.magento.com/guides/v2.2/cloud/before/before-workspace-ssh.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by jeff-matthews** [comment]
Run tests

**Comment by osrecio** [comment]
@jeff-matthews can you run tests, again? 

The changes made here are ok for me

File: _includes/install/file-system-perms-twouser.md [hunk 1]
```diff
@@ -45,9 +45,13 @@ Because the point of creating this user is to provide added security, make sure
 
 To find the web server user's group:
 
-*	CentOS: `egrep -i '^user|^group' /etc/httpd/conf/httpd.conf`
+*	CentOS: 
 
-	Typically, the user and group name are both `apache`
+	grep -E -i '^user|^group' /etc/httpd/conf/httpd.conf
```
Comment (jeff-matthews) [inline _includes/install/file-system-perms-twouser.md]:
```suggestion
    ```bash
    grep -E -i '^user|^group' /etc/httpd/conf/httpd.conf
    ```
```

File: _includes/install/file-system-perms-twouser.md [hunk 2]
```diff
@@ -45,9 +45,13 @@ Because the point of creating this user is to provide added security, make sure
...
+    # or
```
Comment (jeff-matthews) [inline _includes/install/file-system-perms-twouser.md]:
```suggestion
    or
```

File: _includes/install/file-system-perms-twouser.md [hunk 3]
```diff
@@ -45,9 +45,13 @@ Because the point of creating this user is to provide added security, make sure
...
+    grep -Ei '^user|^group' /etc/httpd/conf/httpd.conf
```
Comment (jeff-matthews) [inline _includes/install/file-system-perms-twouser.md]:
```suggestion
    ```bash
    grep -Ei '^user|^group' /etc/httpd/conf/httpd.conf
    ```
```

File: _includes/install/file-system-perms-twouser_22.md [hunk 4]
```diff
@@ -45,9 +45,14 @@ Because the point of creating this user is to provide added security, make sure
 
 To find the web server user's group:
 
-*	CentOS: `egrep -i '^user|^group' /etc/httpd/conf/httpd.conf`
+*	CentOS:
+
+	grep -E -i '^user|^group' /etc/httpd/conf/httpd.conf
```
Comment (jeff-matthews) [inline _includes/install/file-system-perms-twouser_22.md]:
```suggestion
    ```bash
    grep -E -i '^user|^group' /etc/httpd/conf/httpd.conf
    ```
```

File: _includes/install/file-system-perms-twouser_22.md [hunk 5]
```diff
@@ -45,9 +45,14 @@ Because the point of creating this user is to provide added security, make sure
...
+    # or
```
Comment (jeff-matthews) [inline _includes/install/file-system-perms-twouser_22.md]:
```suggestion
    or
```

File: _includes/install/file-system-perms-twouser_22.md [hunk 6]
```diff
@@ -45,9 +45,14 @@ Because the point of creating this user is to provide added security, make sure
...
+    grep -Ei '^user|^group' /etc/httpd/conf/httpd.conf
```
Comment (jeff-matthews) [inline _includes/install/file-system-perms-twouser_22.md]:
```suggestion
    ```bash
    grep -Ei '^user|^group' /etc/httpd/conf/httpd.conf
    ```
```

File: guides/v2.0/cloud/before/before-workspace-file-sys-owner.md [hunk 7]
```diff
@@ -47,9 +47,13 @@ Because the point of creating this user is to provide added security, make sure
 
 To find the web server user's group:
 
-*	CentOS: `egrep -i '^user|^group' /etc/httpd/conf/httpd.conf`
+*	CentOS: 
 
-	Typically, the user and group name are both `apache`
+	grep -E -i '^user|^group' /etc/httpd/conf/httpd.conf
```
Comment (jeff-matthews) [inline guides/v2.0/cloud/before/before-workspace-file-sys-owner.md]:
```suggestion
    ```bash
    grep -E -i '^user|^group' /etc/httpd/conf/httpd.conf
    ```
```

File: guides/v2.0/cloud/before/before-workspace-file-sys-owner.md [hunk 8]
```diff
@@ -47,9 +47,13 @@ Because the point of creating this user is to provide added security, make sure
...
+    # or
```
Comment (jeff-matthews) [inline guides/v2.0/cloud/before/before-workspace-file-sys-owner.md]:
```suggestion
    or
```

File: guides/v2.0/cloud/before/before-workspace-file-sys-owner.md [hunk 9]
```diff
@@ -47,9 +47,13 @@ Because the point of creating this user is to provide added security, make sure
...
+    grep -Ei '^user|^group' /etc/httpd/conf/httpd.conf
```
Comment (jeff-matthews) [inline guides/v2.0/cloud/before/before-workspace-file-sys-owner.md]:
```suggestion
    ```bash
    grep -Ei '^user|^group' /etc/httpd/conf/httpd.conf
    ```
```

File: guides/v2.1/cloud/before/before-workspace-file-sys-owner.md [hunk 10]
```diff
@@ -47,9 +47,13 @@ Because the point of creating this user is to provide added security, make sure
 
 To find the web server user's group:
 
-*	CentOS: `egrep -i '^user|^group' /etc/httpd/conf/httpd.conf`
+*	CentOS: 
 
-	Typically, the user and group name are both `apache`
+	grep -E -i '^user|^group' /etc/httpd/conf/httpd.conf
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/before/before-workspace-file-sys-owner.md]:
```suggestion
    ```bash
    grep -E -i '^user|^group' /etc/httpd/conf/httpd.conf
    ```
```

File: guides/v2.1/cloud/before/before-workspace-file-sys-owner.md [hunk 11]
```diff
@@ -47,9 +47,13 @@ Because the point of creating this user is to provide added security, make sure
...
+    # or
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/before/before-workspace-file-sys-owner.md]:
```suggestion
    or
```

File: guides/v2.1/cloud/before/before-workspace-file-sys-owner.md [hunk 12]
```diff
@@ -47,9 +47,13 @@ Because the point of creating this user is to provide added security, make sure
...
+    grep -Ei '^user|^group' /etc/httpd/conf/httpd.conf
```
Comment (jeff-matthews) [inline guides/v2.1/cloud/before/before-workspace-file-sys-owner.md]:
```suggestion
    ```bash
    grep -Ei '^user|^group' /etc/httpd/conf/httpd.conf
    ```
```

**Comment by jeff-matthews** [comment]
@gelanivishal, I made several suggestions to ensure the formatting meets our standards. You should accept each suggestion so I can continue processing your PR.

**Comment by gelanivishal** [comment]
@jeff-matthews It's done now.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**